docs/source/optim.rst

```diff
@@ -146,6 +146,35 @@ allows dynamic learning rate reducing based on some validation measurements.
 Learning rate scheduling should be applied after optimizer's update; e.g., you
 should write your code this way:
 
+Example::
+
+    model = [Parameter(torch.randn(2, 2, requires_grad=True))]
+    optimizer = SGD(model, 0.1)
+    scheduler = ExponentialLR(optimizer, gamma=0.9)
+
+    for epoch in range(20):
+        optimizer.step()
+        scheduler.step()
```

Comment:
Observations:
- It's easier to compare different sections of the documentation if they are as similar as possible.
- It's important to tell users where the rest of the training code would go relative to the scheduler step.
- The optimizer is stepped at every dataset iteration, not just epoch.

This is a little verbose but bridges the two types of examples presented on the documentation page.
```
    model = [Parameter(torch.randn(2, 2, requires_grad=True))]
    optimizer = SGD(model, 0.1)
    scheduler = ExponentialLR(optimizer, gamma=0.9)

    for epoch in range(20):
        for input, target in dataset:
            optimizer.zero_grad()
            output = model(input)
            loss = loss_fn(output, target)
            loss.backward()
            optimizer.step()
        scheduler.step()
```

Reply:
Yes, indeed this looks much more consistent with the rest of the documentation :)

To preserve consistency even more i modified chain example as well accordingly. Hope it was a right decision :)